README.md

```diff
@@ -94,7 +94,7 @@ docker push 934349674827.dkr.ecr.us-east-1.amazonaws.com/cognoma-core-service:la
 
 Run these commands:
 ```
-docker build -t cognoma-nginx -f config/prod/Dockerfile_nginx .
+docker build -tag cognoma-nginx -file config/prod/Dockerfile_nginx .
```

Comment:
Yes of course! Whoops.

I definitely agree that programs that use single hyphens for long args are trolls though haha